delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/indexing/BlazegraphIndexingCoordinator.scala

```diff
@@ -0,0 +1,167 @@
+import ch.epfl.bluebrain.nexus.sourcing.projections.{Projection, ProjectionId, ProjectionProgress}
+import com.typesafe.scalalogging.Logger
+import fs2.Stream
+import monix.bio.Task
+import monix.execution.Scheduler
+import retry.CatsEffect._
+import retry.syntax.all._
+
+object BlazegraphIndexingCoordinator {
+
+  private val logger: Logger = Logger[BlazegraphIndexingCoordinator.type]
+
+  def apply(
+      views: BlazegraphViews,
+      eventLog: GlobalEventLog[ResourceF[ExpandedJsonLd]],
+      client: BlazegraphClient,
+      projection: Projection[Unit],
+      config: BlazegraphViewsConfig
+  )(implicit
+      as: ActorSystem[Nothing],
+      scheduler: Scheduler,
+      base: BaseUri
+  ): Task[IndexingStreamCoordinator[ResourceF[IndexingBlazegraphView], Unit]] = {
+    def idF(viewRes: ResourceF[IndexingBlazegraphView]): ViewProjectionId = ViewProjectionId(
+      s"blazegraph-indexer-${viewRes.value.project}-${viewRes.value.id}"
+    )
+
+    def buildStream(
+        viewRes: ResourceF[IndexingBlazegraphView],
+        initialProgress: ProjectionProgress[Unit]
+    ): Task[Stream[Task, Unit]] = {
+      implicit val projectionId: ProjectionId = idF(viewRes)
+      val view                                = viewRes.value
+      val namespace                           = s"${config.client.indexPrefix}_${view.project}_${view.uuid}_${viewRes.rev}"
+      val retryStrategy                       = RetryStrategy[SparqlClientError](
+        config.client.retry,
+        {
+          case WrappedHttpClientError(_) => true
+          case _                         => false
+        },
+        logError(logger, "blazegraph client")
+      )
+      import retryStrategy.{errorHandler, policy, retryWhen}
+      for {
+        _     <- client
+                   .createNamespace(namespace, Map.empty)
+                   .retryingOnSomeErrors(retryWhen)
+                   .hideErrorsWith(err => new IllegalArgumentException(err.toString()))
+        eLog  <- eventLog
+                   .stream(view.project, initialProgress.offset, view.resourceTag)
+                   .hideErrorsWith(e => new IllegalArgumentException(e.reason))
+        stream = eLog
+                   .filterMessage(res =>
+                     (view.resourceTypes.isEmpty || res.types
+                       .union(view.resourceTypes)
+                       .nonEmpty)
+                       && (view.resourceSchemas.isEmpty || view.resourceSchemas.contains(
+                         res.schema.iri
+                       )) && (view.includeDeprecated || res.deprecated)
+                   )
+                   .flatMapMessage { msg =>
+                     msg.value.value.toGraph
+                       .map { graph =>
+                         if (view.includeMetadata) {
+                           val types: Set[Triple] = msg.value.types.map((msg.value.id, rdf.tpe, _))
+                           graph
+                             .add(types)
```

Comment:
the triples coming from the URI are missing:
- nxv.project
- nxv.outgoing
- nxv.incoming